app/src/main/java/org/mifos/mobile/ui/activities/SplashActivity.java

```diff
@@ -5,13 +5,27 @@
  */
 
 import android.content.Intent;
+import android.content.pm.ShortcutInfo;
```

Comment:
remove the following imports as these aren't being used :- 8, 9 , 10, 12, 18, 19, 20, 21, 22, 23, 26, 27 (these are line numbers)